.github/workflows/android_deploy_production.yml

```diff
@@ -58,4 +59,5 @@ jobs:
           appId: ${{ vars.FIREBASE_ANDROID_APP_ID }}
           serviceCredentialsFileContent: ${{ secrets.FIREBASE_DISTRIBUTION_CREDENTIAL_JSON }}
           groups: ${{ vars.FIREBASE_DISTRIBUTION_TESTER_GROUPS }}
+          releaseNotes: ${{ env.RELEASE_NOTE }}
```

Comment:
Hmm, where does this env come from? Do you mean it will be generated from the step `Set up .env`? Is it `RELEASE_NOTE_CONTENT` instead?